### PR TITLE
adjusting client to embed-changes ushahidi/platform#2457

### PR DESCRIPTION
**Pull request by Angamanga**
This pull request makes the following changes:
- Adds the mode-bar and filter-bar to embeds that are bigger than 1023px
- Fixes problems with data-view and embeds

Testing checklist:
-  Look at the client as an embed for /views/map with a width smaller than 1023px
- [ ] No modebar or the 'normal' filter-bar should be visible
-  Click on a post to get to the detailed view
- [ ] You should see the post with a x in the upper right corner
- [ ] Still no modebar or filter-bar
- Click on the x
- [ ] You should be directed back to the map-view
-  Look at the client as an embed for /views/data with a width smaller than 1023px
- [ ] No modebar or the 'normal' filter-bar should be visible
- [ ] You should see the postlist only
- Click on a post
- [ ] You should see the post with a x in the upper right corner
- [ ] Still no modebar or filter-bar
- [ ] Click on the x
- [ ] You should be directed back to the post-list

-  Look at the client as an embed for /views/map with a width larger than 1023px
- [ ] the modebar and the 'normal' filter-bar should be visible
-  Click on a post to get to the detailed view
- [ ] You should see the post-list to the left and the post on the right
- [ ] Modebar and filter-bar is visible

Click on data in the navigation
- [ ] You should see the post-list to the left and the post on the right
- [ ] Modebar and filter-bar is visible

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by tuxpiper** [comment]
It looks like we may be having problems including the pattern library, during the integration build. Browser console reports: "Error: Cannot find module "ushahidi-platform-pattern-library/assets/css/style.min.css"

So cannot really test this yet.

Investigating the problem

**Comment by tuxpiper** [comment]
@Angamanga 

I've published the PL branch to npmjs.org and changed the reference to it from package.json . It's a bit annoying that it's designed to work like that, but it's the way it is.. platform-client will only work well when installing the PL from npmjs.org

**Comment by Angamanga** [comment]
Oh, sorry @tuxpiper It's still wip since the PL-pr is not merged yet... Should have written WIP.  Thanks for releasing, hopefully no more commits to add to the pl :)

File: app/main/posts/views/post-view-data.html [hunk 1]
```diff
@@ -1,4 +1,11 @@
+<div class="mode-context init" embed-only="true" ng-controller="navigation as nav" dropdown>
+    <header class="mode-context-header" ng-class="{ 'has-logo' : nav.site.image_header }">
+      <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
+      <img ng-if="nav.site.image_header" ng-src="{{ nav.site.image_header }}" class="deployment-logo" />
+    </header>
```
Comment (Angamanga) [inline app/main/posts/views/post-view-data.html]: Above is ugly, but I couldn't find a way to add it somewhere else. Its just to get the deployment-header in data-view in embeds.

**Comment by Angamanga** [comment]
For testing, you can save below text into a file and save it as 'index.html', and then open it in a browser:
``` 
<h3>Map-view with the default embed size</h3>
<iframe width="560" height="315" src="http://qa.2454-embeds.ushahidilab.com/views/map" frameborder="0" allowfullscreen></iframe>
<h3>Data-view with the default embed size</h3>
<iframe width="560" height="315" src="http://qa.2454-embeds.ushahidilab.com/views/data" frameborder="0" allowfullscreen></iframe>
<h3>Map-view for larger screens</h3>
<iframe width="1030" height="700" src="http://qa.2454-embeds.ushahidilab.com/views/map" frameborder="0" allowfullscreen></iframe>
<h3>Data-view for larger screens</h3>
<iframe width="1030" height="700" src="http://qa.2454-embeds.ushahidilab.com/views/data" frameborder="0" allowfullscreen></iframe>
```

**Comment by tuxpiper** [comment]
@Angamanga  Is the share button looking a bit funny?

screen shot 2018-02-07 at 14 57 54

**Comment by Angamanga** [comment]
@tuxpiper sorry, seems like I never pushed my last changes, doing it now!

**Comment by tuxpiper** [comment]
ref ushahidi/platform#2457  ushahidi/account-management#2

**Comment by tuxpiper** [comment]
what are the next steps here? @Angamanga @Eve-wanderi  . Has this been QA'ed?

**Comment by Angamanga** [comment]
@tuxpiper I am waiting for @Eve-wanderi to finish QA, will demo to romina once too when she wakes up just as a security-measure. I'll let you know when its ready to go to production!

**Comment by tuxpiper** [comment]
Great thank you! Let's keep in mind that we promised to deliver by Monday

**Comment by Angamanga** [comment]
Note: I messed up the PL-branch so need to fix that before deploying/qa-ing any further

**Comment by tuxpiper** [comment]
I approve (again)

**Pull request merged**